.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,13 +6,18 @@ Fixes #
 - [ ] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
 - [ ] My branch is up-to-date with the Upstream `master` branch.
 - [ ] I have added necessary documentation (if appropriate)
+- [ ] Added Surge preview link
```

Comment:
@YashKumarVerma Add the commented block here `Replace "PR_NUMBER" with your pull request number. This link is generated when your PR passes the travis tests.A sample link can look like https://pr-200-fossasia-susper.surge.sh`